reinstall.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
We could add `set -e` at the top to ensure the script stops if a command fails, preventing half-completed installations.
```
# Exit immediately if a command exits with a non-zero status
set -e
```
Not a must just a suggestion :)